Makefile

```diff
@@ -39,15 +39,13 @@ $(shell mkdir -p ./out);
 # HELP target
 #----------------------------------------------------------------
 
-## This target is used to display all other target
-.PHONY: help
 # Based on https://gist.github.com/rcmachado/af3db315e31383502660
 ## Display this help text
 help:/
 	$(info Available targets)
 	$(info -----------------)
 	@awk '/^[a-zA-Z\-%\_0-9]+:/ { \
-		helpMessage = match(lastLine, /^## (.*)/); \
+		helpMessage = match(lastLine, /^## (.*)/);
```

Comment:
We need `\` here otherwise it will give `unexpected EOF while looking for matching `''` error.

I'm trying to run `make help` throwing error:
```
Available targets
-----------------
/bin/bash: -c: line 0: unexpected EOF while looking for matching `''
/bin/bash: -c: line 2: syntax error: unexpected end of file
make: *** [Makefile:47: help] Error 1
```